cmake/Modules/FindOpenSSL.cmake

```diff
@@ -32,3 +32,11 @@ else()
   endif()
 endif()
 
+# OpenSSL DLL on Windows: use of BIO_s_fd and BIO_s_file (directly or indirectly) requires
+# the executable to incorporate OpenSSL applink.c.  CMake 18 adds support for handling
```

Comment:
Maybe post a link to that in the PR? If it's a module, we could opt to copy that module in and override. Just an idea.

Reply:
This is what I saw: https://cmake.org/cmake/help/v3.18/module/FindOpenSSL.html

Given that we have some hacks to deal with Conan-provided OpenSSL already and that this works reliably on the platforms we support, I think it makes the most sense to merge it as-is and deal with those hacks later. That way the ROS 2 CI problems should be addressed.

Since you approved it already, I'm sure you're ok with that :)